pkg/model/const.go

```diff
@@ -126,6 +126,11 @@ const (
 	RemoteSubPath = "okteto-remote"
 	// OktetoAutoCreateAnnotation indicates if the deployment was auto generated by okteto up
 	OktetoAutoCreateAnnotation = "dev.okteto.com/auto-create"
+	// OktetoDivertedSourceAnnotation indicates the namespace source of the diverted resource (usually a service or an ingress)
+	// The value of this annotation is the namespace of the original resource
+	OktetoDivertedSourceAnnotation = "divert.okteto.com/source-namespace"
```

Comment:
To keep it shorter and consistent with the okteto manifest divert section, I would rename this one to `divert.okteto.com/namespace`